pilot/cmd/pilot-discovery/main.go

```diff
@@ -35,6 +40,15 @@ import (
 	"istio.io/pkg/version"
 )
 
+const (
```

Comment:
I think this should go under ServerOpts, and the code in bootstrap.go, not main.

Reply:
Done.